rest/taskana-rest-spring/src/main/java/pro/taskana/common/rest/JsonPropertyEditor.java

```diff
@@ -0,0 +1,27 @@
+package pro.taskana.common.rest;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.beans.PropertyEditorSupport;
+import java.net.URLDecoder;
+
+public class JsonPropertyEditor extends PropertyEditorSupport {
+
+  private final ObjectMapper objectMapper;
+  private final Class<?> requiredType;
+
+  public JsonPropertyEditor(ObjectMapper objectMapper, Class<?> requiredType) {
+    this.objectMapper = objectMapper;
+    this.requiredType = requiredType;
+  }
+
+  @Override
+  public void setAsText(String text) throws IllegalArgumentException {
+    if (text != null && !text.isEmpty()) {
+      try {
+        setValue(objectMapper.readValue(URLDecoder.decode(text, "UTF-8"), requiredType));
```

Comment:
this additional decoding of the text is not required. For some reason our tests fail, because `text` is not decoded by spring.. Therefore I added this additional decoding. It does not hurt.